client/components/seo-preview-pane/index.jsx

```diff
@@ -117,25 +161,29 @@ export class SeoPreviewPane extends PureComponent {
 						</p>
 					</div>
 					<VerticalMenu onClick={ this.selectPreview }>
-						<SocialItem service="google" />
-						<SocialItem service="facebook" />
-						<SocialItem service="twitter" />
+						{ compact( [
+							post && 'wordpress',
+							'google',
+							'facebook',
+							'twitter'
+						] ).map( service => <SocialItem { ...{ key: service, service } } /> )
+						}
```

Comment:
I'm more in favor of extracting this into a variable I think, just because of the awkwardness here…

```js
const services = compact( [
	post && 'wordpress',
	'google',
	'facebook',
	'twitter'
] );

…

{ services.map( service => <SocialItem { ...{ key: service, service } } /> }
```